01-JS-DOM-APIs/12.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html class="html">
+<head>
+  <link href="style 12.css" rel="stylesheet">
```

Comment:
**minor:** try to avoid naming files with spaces such as *style 12.css* as this could cause unexpected behaviors